### ISSUE TITLE
Result is different from paper 

### ISSUE THREAD

**Issue by zou-yawen** [vimar-gu/MinimaxDiffusion#4]
hi, I  git clone the code and run it.  But the result is 59.4% different from the paper could you please help me to check it?
the data is imagewoof download from [Full size](https://s3.amazonaws.com/fast-ai-imageclas/imagewoof2.tgz) and the code follows below:
torchrun --nnode=1 --master_port=25678 train_dit.py --model DiT-XL/2      --data-path imagewoof2/train/ --ckpt DiT-XL-2-256x256.pt      --global-batch-size 8 --tag minimax --ckpt-every 500 --log-every 1500 --epochs 8      --condense --finetune-ipc -1 --results-dir ../logs/run-0 --spec woof
python sample.py --model DiT-XL/2 --image-size 256 --ckpt ../logs/run-0/010-DiT-XL-2-minimax/checkpoints/0009000.pt     --save-dir ../results/dit-distillation/imagenet-10-1000-minimax --spec woof
python train.py -d imagenet --imagenet_dir ../results/dit-distillation/imagenet-10-1000-minimax imagewoof2/     -n resnet_ap --nclass 10 --norm_type instance --ipc 100 --tag test --slct_type random --spec woof

**Comment by vimar-gu** [comment]
Thanks for the interest in this work! 
I used the ImageWoof classes from the original ImageNet dataset, which contains more images than the dataset in the fastai repo. Please try using the original data and running the experiment again :)

**Comment by zou-yawen** [comment]
> Thanks for the interest in this work! I used the ImageWoof classes from the original ImageNet dataset, which contains more images than the dataset in the fastai repo. Please try using the original data and running the experiment again :)

Thank you very much for your quick reply. I will try it.

**Comment by zou-yawen** [comment]
> > Thanks for the interest in this work! I used the ImageWoof classes from the original ImageNet dataset, which contains more images than the dataset in the fastai repo. Please try using the original data and running the experiment again :)
> 
> Thank you very much for your quick reply. I will try it.
could you please give me a link to download the dataset same as yours, I find there are so many versions of the ImageNet dataset.

**Comment by vimar-gu** [comment]
Recently I found that downloading ImageNet from huggingface is quite fast. You can refer to [this link](https://huggingface.co/datasets/ILSVRC/imagenet-1k). 

You can also use [torrents](https://academictorrents.com/collection/imagenet) for p2p downloading. 

I have tried both. Torrent downloading yields the original dataset structure. And huggingface downloading is convenient for servers.

**Comment by zou-yawen** [comment]
Thank you very much. I will try it.

**Comment by zou-yawen** [comment]
> Recently I found that downloading ImageNet from huggingface is quite fast. You can refer to [this link](https://huggingface.co/datasets/ILSVRC/imagenet-1k).
> 
> You can also use [torrents](https://academictorrents.com/collection/imagenet) for p2p downloading.
> 
> I have tried both. Torrent downloading yields the original dataset structure. And huggingface downloading is convenient for servers.

Thank you very much. I repeated the experiments and obtained good results: 65.2 (seed=0), 64.8 (seed=1), and 62 (seed=2). The average accuracy is 64, which is similar to the result reported in the paper.

**Comment by vimar-gu** [comment]
Thanks for letting me know 😄
